pac4j-core/src/main/java/org/pac4j/core/profile/converter/GenderConverter.java

```diff
@@ -1,41 +1,41 @@
 package org.pac4j.core.profile.converter;
 
+import java.util.regex.Pattern;
+
 import org.pac4j.core.profile.Gender;
 
 /**
  * This class converts a String to a Gender.
- * 
+ *
  * @author Jerome Leleu
  * @since 1.0.0
  */
 public final class GenderConverter extends AbstractAttributeConverter<Gender> {
 
+    private final Pattern maleText;
+    private final Pattern femaleText;
+
     public GenderConverter() {
         super(Gender.class);
+        this.maleText = Pattern.compile("(^m$)|(^male$)");
+        this.femaleText = Pattern.compile("(^f$)|(^female$)");
```

Comment:
Why not add here the "1" pattern for the female gender? Something like: `(^f$)|(^female$)|(^1$)`

Reply:
1 and 2 may be different in different situations. For example, wechat uses 1 for male and 2 for female.